test/io-tests/test_io.py

```diff
@@ -0,0 +1,175 @@
+'Tests for inputs and outputs of PostgREST.'
+
+import contextlib
+import pathlib
+import subprocess
+import tempfile
+import os
+import time
+
+import pytest
+import requests
+
+BASEURL = 'http://127.0.0.1:49421'
+
+
+basedir = pathlib.Path(os.path.realpath(__file__)).parent
+configs = [path for path in (basedir / 'configs').iterdir() if path.is_file()]
+expectedconfigs = list((basedir / 'configs' / 'expected').iterdir())
+secrets = [path for path in (basedir / 'secrets').iterdir() if path.suffix != '.jwt']
+dburi = os.getenv('POSTGREST_TEST_CONNECTION')
+dburifromfileconfig = basedir / 'configs' / 'dburi-from-file.config'
+roleclaimkeyconfig = basedir / 'configs' / 'role-claim-key.config'
+
+invalidroleclaimkeys = [
+        'role.other',
```

Comment:
What do you think about 2 spaces instead of 4 spaces indentation across the whole python file? I think that's what we're using across the other files in the project.

Reply:
Ah, I was going to add automatic formatting with e.g. [black](https://black.readthedocs.io/en/stable/) (just added in last commit), which enforces the Python standard of [four spaces](https://www.python.org/dev/peps/pep-0008/#indentation). For Python code I think it's better when we stick to that standard, and postgrest-style will now make sure that we stick to that no matter how contributors add new code. WDYT?

Reply:
I don't think I would have even mentioned it, if it wasn't for the 8 spaces that the array items had in the spot I commented on. I see black has made that 4 spaces only by pulling the `]` out one level. This satisfies my eye enough - I won't object to this :).